bin/eshost.js

```diff
@@ -63,11 +63,18 @@ const yargv = yargs
   .nargs('delete', 1)
   .describe('args', 'set arguments for a host entry (use with --add)')
   .nargs('args', 1)
+  .describe('configure-jsvu', 'Configure JSVU hosts in the config')
+  .boolean('configure-jsvu')
+  .describe('jsvu-root', 'Use this path containing the .jsvu folder (use this option if .jsvu is located somewhere other than the home directory).')
+  .nargs('jsvu-root', 1)
+  .describe('jsvu-prefix', 'Set the prefix of the configured hosts. If prefix is "jsvu" then hosts will be configured as, e.g., "jsvu-sm". By default, no prefix (e.g. "sm").')
+  .nargs('jsvu-prefix', 1)
   .help('help')
   .example('eshost --list')
   .example('eshost --add d8 d8 path/to/d8 --args "--harmony"')
   .example('eshost --add ch ch path/to/ch --tags latest')
   .example('eshost --add ch ch path/to/ch --tags latest,greatest')
+  .example('eshost --configure-jsvu --jsvu-root /my/jsvu/root --jsvu-prefix jsvu')
```

Comment:
The jsvu dir is always located at `~/.jsvu`. Can we update the example accordingly, or remove the `--jsvu-root` flag (or make it optional, with `~/.jsvu` as the default value)?

Reply:
It is optional. I'll add more examples to clarify.